### ISSUE TITLE
Autocorrect for Rails/Presence corrupts code when there's an `elsif` block

### ISSUE THREAD

**Issue by ivgiuliani** [bbatsov/rubocop#5250]
## Expected behavior

I would expect rubocop to leave this code unchanged

## Actual behavior

The autocorrect is corrupting the surrounding code, by misplacing the `elsif` clause

## Steps to reproduce the problem

```
$ cat test.rb
```
```ruby
class TestModel < ActiveRecord::Base
  def foo?
    attr1.present? || attr2.present?
  end

  def testcase
    if attr3.present?
      attr3
    elsif foo?
      "something"
    else
      raise "Invalid"
    end
  end
end
```

```
$ rubocop -a --only "Rails/Presence" ./test.rb
Inspecting 1 file
E

Offenses:

test.rb:9:5: C: [Corrected] Rails/Presence: Use attr3.presence || elsif foo?
      "something"
    else
      raise "Invalid" instead of if attr3.present?
      attr3
    elsif foo?
      "something"
    else
      raise "Invalid"
    end.
    if attr3.present? ...
    ^^^^^^^^^^^^^^^^^
test.rb:9:23: E: Lint/Syntax: unexpected token kELSIF
(Using Ruby 2.4 parser; configure using TargetRubyVersion parameter, under AllCops)
    attr3.presence || elsif foo?
                      ^^^^^
test.rb:11:5: W: Lint/Syntax: else without rescue is useless
(Using Ruby 2.4 parser; configure using TargetRubyVersion parameter, under AllCops)
    else
    ^^^^

1 file inspected, 3 offenses detected, 1 offense corrected
```

```
$ cat ./test.rb
```
```ruby
class TestModel < ActiveRecord::Base
  def foo?
    attr1.present? || attr2.present?
  end

  def testcase
    attr3.presence || elsif foo?
      "something"
    else
      raise "Invalid"
  end
end
```

## RuboCop version

`0.52.0 (using Parser 2.4.0.2, running on ruby 2.4.2 x86_64-darwin16)`

**Comment by mensfeld** [comment]
@bbatsov  It's not only that. The inspection itself is broken as well when running rails cops for the case mentioned by @ivgiuliani :

```
An error occurred while Rails/Presence cop was inspecting (blablabla)
```

**Comment by jfelchner** [comment]
I"m not sure if this case was caught, but this cop should skip if the `else` block contains more than one statement.
